### DISCUSSION TITLE
Tailwind CSS IntelliSense?

### DISCUSSION THREAD

**Discussion by CharlieBytesX** [rustshop/htmx-sorta#15] [Q&A] [answered]
Hey did you manage to get tailwinds intellisense - lsp for your editor inside mau macros?
If so, could you share how?

**Comment by CharlieBytesX** [comment]
I managed to make it work in vscode with these settings:

```json
 "tailwindCSS.includeLanguages": {
    "rust": "html",
  },
```
 
And created a regex for capturing this pattern:
anyword. ""

```json
"tailwindCSS.experimental.classRegex": [
  [
    ".*\\.\\s*?[\"']([^\"'`]*).*?"
  ],
],
```
  
<img width="1552" alt="image" src="https://github.com/rustshop/htmx-sorta/assets/62361574/1ba40226-05e6-4c7a-9401-1df1a8f455f0">

**Reply by nik-here** [reply, marked as answer]
Add this:

```
"tailwindCSS.experimental.classRegex": [
  [
    ".*\\.\\s*?[\"']([^\"'`]*).*?",
    "class=[\"']([^\"']*)"
  ],
],
```
For 'class="..."' syntaxt.
E.g:
```
html ! {
    div class="px-4" {
        "foo"
    }
}
```

**Comment by CharlieBytesX** [comment]
Ups, I thought that mau only accepts the "." notation, thanks a lot!!
